### PR TITLE
Adding intellisense for the "Title" property, which is used by NuGet Packaging.

### PR DESCRIPTION
**Pull request by tannergooding**
This resolves https://github.com/dotnet/project-system/issues/1800

### CODE REVIEW

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 1]
```diff
@@ -1774,6 +1774,11 @@ elementFormDefault="qualified">
         <xs:documentation><!-- _locID_textPackageId _locComment="" -->The case-insensitive NuGet package identifier, which must be unique across nuget.org or whatever gallery the NuGet package will reside in. IDs may not contain spaces or characters that are not valid for a URL, and generally follow .NET namespace rules.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="Title" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+      <xs:annotation>
+        <xs:documentation><!-- _locID_textPackageId _locComment="" -->A human-friendly title of the package, typically used in UI displays as on nuget.org and the Package Manager in Visual Studio. If not specified, the package ID is used instead.</xs:documentation>
```
Comment (rainersigwald) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Change `_locID_textPackageId` to `_locID_text="Title"` in the comment, please.

**Comment by rohit21agrawal** [comment]
LGTM

File: src/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (tannergooding) [inline src/MSBuild/Microsoft.Build.CommonTypes.xsd]: Fixed.

**Comment by tannergooding** [comment]
Just for reference, the wording was taken directly from the NuSpec documentation page: https://docs.microsoft.com/en-us/nuget/schema/nuspec

**Comment by rainersigwald** [comment]
@dotnet-bot test Windows_NT Build for Desktop please

**Comment by rainersigwald** [comment]
Thanks, @tannergooding.

**Pull request merged**